kevm

```diff
@@ -259,8 +259,8 @@ run_solc() {
     contract_name="$1" ; shift
     json_run_file="${run_file}.json"
 
-    notif_exec solc --combined-json abi,bin-runtime,storage-layout "$@" "${run_file}" > "${json_run_file}"
-    notif_exec kevm-pyk.py solc-to-k ${backend_dir}/*-kompiled "${run_file}" "${json_run_file}" "${contract_name}"
+    notif_exec solc --combined-json abi,bin-runtime,storage-layout "${run_file}" > "${json_run_file}"
+    notif_exec python3 -m kevm_pyk solc-to-k "$@" ${backend_dir}/*-kompiled "${run_file}" "${json_run_file}" "${contract_name}"
```

Comment:
For easy passing of keyword arguments to `kevm_pyk`, I moved `$@` from the invocation of `solc` to the invocation of `kevm_pyk`. Is it safe, or do we need to call `solc` with optional arguments as well? (In this case, we will have to distinguish between these two types of arguments.)